data/migrations/20190116125500_users_table.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('users', users =>{
+        users.increments();
+
+        users
+            .string('username', 128)
+            .notNullable()
+            .unique();
```

Comment:
Good job adding unique as a users property to verify that no username is used twice 👍